docs/api/navigators/StackNavigator.md

```diff
@@ -137,7 +137,7 @@ Color for material ripple (Android >= 5.0 only)
 
 #### `gesturesEnabled`
 
-Whether you can use gestures to dismiss this screen. Defaults to true on iOS, false on Android
+Whether you can use gestures to dismiss this screen. Defaults to true on iOS, false on Android. This value when set is only respected in `mode` == `card`.
```

Comment:
My fault, but I think an extra `StackNavigatorConfig` would be much better, on Android, gestures are disabled by default, I have to add `gesturesEnabled` to all the screens in the stack to make them have the same behavior on both iOS and Android, I think that's insane, so I propose we could get the default value from `StackNavigatorConfig`